Index: src/main/java/com/myblog/myblog11/controller/PostController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/myblog/myblog11/controller/PostController.java b/src/main/java/com/myblog/myblog11/controller/PostController.java
--- a/src/main/java/com/myblog/myblog11/controller/PostController.java	(revision 91e1e8228a54f6079056b57c15dfa6bfd96addbd)
+++ b/src/main/java/com/myblog/myblog11/controller/PostController.java	(revision 3771faa5c78f5c26d7ca9301e4aa3bc1807eddc0)
@@ -19,7 +19,7 @@
         this.postService = postService;
     }
 
-    @PostMapping
+    @PostMapping("/")
     public ResponseEntity<PostDto> createPost(@RequestBody PostDto postDto){
         PostDto dto = postService.createPost(postDto);
         return new ResponseEntity <> (dto, HttpStatus.CREATED);
