Index: src/main/java/com/myblog/myblog11/entity/post.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/myblog/myblog11/entity/post.java b/src/main/java/com/myblog/myblog11/entity/post.java
new file mode 100644
--- /dev/null	(revision ca8ea434b45bff6ba40dc81f2e67c3167dce185d)
+++ b/src/main/java/com/myblog/myblog11/entity/post.java	(revision ca8ea434b45bff6ba40dc81f2e67c3167dce185d)
@@ -0,0 +1,28 @@
+package com.myblog.myblog11.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.*;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static jakarta.persistence.GenerationType.AUTO;
+
+
+//use @data from lambok automatically write getters and setters
+//@AllArgsConstructor uses for create auto constroctors from lambok
+//@NoArgsConstructor no arguments constroctors from lambok
+@Data
+@Entity
+@Table(name="posts")
+@AllArgsConstructor
+@NoArgsConstructor
+public class post {
+    @Id
+    @GeneratedValue(strategy = AUTO)
+    private long id;
+    private String title;
+    private String description;
+    private String content;
+}
